TripPlanner/TripPlanner.Controllers/Controllers/BusinessTripRequestController.cs

```diff
@@ -1,25 +1,31 @@
 ﻿using AutoMapper;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
 using TripPlanner.ApiModels.ApiModels;
 using TripPlanner.DatabaseModels.Models;
 using TripPlanner.Logic.Abstractions;
 using TripPlanner.Logic.DtoModels;
+using TripPlanner.Logic.Services.EmailService;
+using TripPlanner.Logic.Services.EmailService.Smtp;
 
 namespace TripPlanner.Controllers.Controllers
 {
-    [Authorize]
+    //[Authorize]
     [ApiController]
     [Route("api/[controller]")]
     public class BusinessTripRequestController : ControllerBase
     {
         private readonly IBusinessTripRequestRepository _repository;
         private readonly IMapper _mapper;
-
-        public BusinessTripRequestController(IBusinessTripRequestRepository repository, IMapper mapper)
+        private readonly IEmailService _emailService;
+        
+        public BusinessTripRequestController(IBusinessTripRequestRepository repository, IMapper mapper , IEmailService emailService)
         {
             _repository = repository;
             _mapper = mapper;
+            _emailService = emailService;
```

Comment:
You should have this dependency in the BusinessTripRequestRepository and call the service after the new trip was created.